### PR TITLE
fix: swap test route path

### LINKED ISSUE #4662: Investigate flaky tests
Several tests are failing intermittently. 

image

This task is to investigate, and share with the team why this happens, as well as recommending improvements how we can avoid in future. API mocking?

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of Leather — [Extension build](https://github.com/leather-wallet/extension/actions/runs/7167418674), [Test report](https://leather-wallet.github.io/playwright-reports/fix/intercept-broadcast-test)

Route path was actually wrong here causing the test to fail. It should be a post request to: `https://sponsor-tx.alexlab.co/v1/graphql`.

### CODE REVIEW

File: tests/specs/swap/swap.spec.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { test } from '../../fixtures/fixtures';
 
-const alexSdkPostRoute = 'https://gql.alexlab.co/v1/graphql';
+const alexSdkPostRoute = '*/**/v1/graphql';
```
Comment (kyranjamie) [inline tests/specs/swap/swap.spec.ts]: Shouldn't this reference the origin at all? Would `*.alexlab.co/v1/graphql` work?

Comment (kyranjamie) [reply tests/specs/swap/swap.spec.ts]: pushing commit for this, think it's good to be specific to origin level at least (who knows maybe that could clash with another api)

**Review by kyranjamie — APPROVED** [review]

File: tests/specs/swap/swap.spec.ts (hunk 1 above)
Comment (kyranjamie) [reply tests/specs/swap/swap.spec.ts]: Tried a few combos before I got it to work while specifying origin

Comment (fbwoolf) [reply tests/specs/swap/swap.spec.ts]: Ok, thx for committing the change.

**Pull request merged**